scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -266,6 +266,7 @@ class ModelDownloadType(Enum):
     FTP = 'ftp'
     GIT = 'git'
     HTTP = 'http'
+    AZURE_BLOBSTORE = 'azure'

```

Comment:
Unless there are other types of Azure storage that we'll need to support, there's no need to distinguish with using the `_BLOBSTORE` suffix. I'd just use `AZURE`.